tests/helpers.py

```diff
@@ -568,3 +568,12 @@ class MockAsyncWebhook(CustomMockMixin, unittest.mock.MagicMock):
     """
     spec_set = webhook_instance
     additional_spec_asyncs = ("send", "edit", "delete", "execute")
+
+
+class MockRedisCache(unittest.mock.MagicMock):
```

Comment:
nit: this should be a NonCallableMagicMock

If an instance of RedisCache can not be called, this shouldn't be callable either.